src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1274,6 +1276,75 @@ public IEnumerable<Tombstone> GetTombstonesFrom(DocumentsOperationContext contex
             }
         }
 
+        public IEnumerable<TombstoneIndexItem> GetTombstoneIndexItemsFrom(DocumentsOperationContext context, long etag, long start, long take)
```

Comment:
this is basically "convert" `DocumentTombstone` to `TombstoneIndexItem` so we could use abstraction and use this generic type for `TimeSeriesDeletedRange` also. TBH, I'm not sure how to handle it otherwise..

Reply:
Cannot the convert part be handled by the cleanup worker? This is how we do it for other `IndexItems`

Reply:
done